x/stake/keeper/validator.go

```diff
@@ -698,7 +686,7 @@ func (k Keeper) RemoveValidator(ctx sdk.Context, address sdk.ValAddress) {
 	store.Delete(GetValidatorsBondedIndexKey(validator.OperatorAddr))
 
 	bz := k.cdc.MustMarshalBinary(validator.ABCIValidatorZero())
-	store.Set(GetTendermintUpdatesKey(address), bz)
+	store.Set(GetTendermintUpdatesTKey(address), bz)
```

Comment:
Should be `tstore.Set()` where `tstore` is `ctx.TransientStore(k.storeTKey)`

Reply:
of course, thank you! - addressed